locale/en/knowledge/command-line/how-to-prompt-for-command-line-input.md

```diff
@@ -11,81 +11,96 @@ layout: knowledge-post.hbs
 
 So you've got a little CLI tool, but you want to be able to prompt a user for additional data after the script has started, rather than passing it in as a command line argument or putting it in a file.  To do this, you'll need to listen to STDIN ("standard input", i.e. your keyboard), which Node.js exposes for you as `process.stdin`, a readable stream.
 
-Streams are Node's way of dealing with evented I/O - they're a big topic, and you can read more about them (here).  For now, we're only going to deal with the Stream methods relevant to working with `process.stdin` so as to keep the examples easy.
-
-The first two Readable Stream methods you'll need to know about here are `pause()` and `resume()`.  Not every program needs to care whether or not you're pressing keys at a given moment, so `process.stdin` is paused by default. 
+Streams are Node's way of dealing with evented I/O - they're a big topic, and you can read more about them [here](https://nodejs.org/api/stream.html). For now, we're going to use node's `readline` module which is a wrapper around Standard I/O, suitable for taking user input from command line(terminal).
```

Comment:
`they're a big topic`：Since you mean the topic is something related to `Streams are Node's way of dealing with evented I/O`, so it may be better to use `it` instead of `they're.